spring-webmvc/src/test/java/org/springframework/web/servlet/mvc/method/annotation/CrossOriginTests.java

```diff
@@ -345,6 +376,34 @@ public void baz() {
 
 	}
 
+	@Target({ElementType.METHOD, ElementType.TYPE})
+	@Retention(RetentionPolicy.RUNTIME)
+	@Documented
+	@CrossOrigin
+	private @interface CrossOriginMetaAnnotation {
```

Comment:
Thanks for the link!
Do you expect a new commit or as you take the issue, are you correcting the issue on your own?